src/client/app/containers/import.ts

```diff
@@ -213,4 +219,36 @@ export class Import {
     cancel() {
         this._store.dispatch(new UI.ToggleImportAction(false));
     }
+
+    checkForViewSnippet() {
+        if (!this.hasViewSettings()) {
+            return;
+        }
+
+        let viewData = Office.context.document.settings.get('VIEW_URL_SETTINGS');
```

Comment:
The view button won't show unless the host is one of Excel, Word, or Powerpoint. So this shouldn't be an issue. Added checks for Office && Office.context via the `isInsideOfficeApp` utility function.